src/Cli/dotnet/Commands/Package/Remove/PackageRemoveCommand.cs

```diff
@@ -69,4 +65,29 @@ private string[] TransformArgs(string packageId, string projectFilePath)
 
         return [.. args];
     }
+
+    private static int ExecuteForFileBasedApp(string path, string packageId)
```

Comment:
Why isn't there CPM logic in the remove operation?

Reply:
That's consistent with the project-based `dotnet package remove` - it leaves the version in Dir.Packages.props because I guess you could be using it from different projects and it would be difficult to detect.